### DISCUSSION TITLE
Things to Improve on Planner

### DISCUSSION THREAD

**Discussion by ZzRanger** [acmutd/comet_planning#215] [Ideas]
Please come up with 3 things to improve with Planner as a group (1 BIG idea and 2 SMALL ideas). Here are some examples of big & small ideas

**Big Ideas** 
1. Integrate Planner with SSO to pull user UTD data
2. Include a way to plan clubs, internships, part-time jobs, etc.

**Small Ideas**

1. Add a "Collect feedback" button
2. Increase the size of the "Add Credits" button

**Comment by ChaoShaoBaoo** [comment]
Big Idea:
1. Incorporate an overarching Selection feature for the Degree Plan(s). 
e.g. Choosing Major, Track, Honors (Req.s), Graduation Time. - these would preset the course order showcase and highlighted "personalized" courses for their preferences. 

Small Ideas:
1. Put +/- signs near semester names to make their function more obvious.
2. Collapse Course List on left side into Subject categories based on beginning abbreviations (e.g. All ACCT classes go into expandable header: ACCT).

**Comment by Afsartech** [comment]
<img width="758" alt="Screen Shot 2022-10-07 at 1 09 09 PM" src="https://user-images.githubusercontent.com/64441916/194622270-90fa960b-03bb-4321-a546-68ff72aea6fd.png">
